<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepEdit</class>
 <widget class="QDialog" name="StepEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/Top.qrc">
    <normaloff>:/new/wl/Logo_ico.png</normaloff>:/new/wl/Logo_ico.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>731</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>339</y>
       <width>731</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pBtnHandAddRask">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>手动增加风险</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>380</y>
      <width>731</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>专家意见列表：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>731</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pBtnRPNSort">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RPN风险排序</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtnShowAll">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>显示所有风险项</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtnFilterTOPRisk">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>过滤TOP(N)风险项</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../qrc/Top.qrc"/>
 </resources>
 <connections/>
</ui>
